### ISSUE TITLE
loss function problem

### ISSUE THREAD

**Issue by hendredlorentz** [leftthomas/SimCLR#32]
Is there no need for l2 regularization in the loss function? This is not the same as the implementation of the (TORCH.NN.FUNCTIONAL.COSINE_SIMILARITY) function on the pytorch official website.

**Comment by leftthomas** [comment]
@hendredlorentz https://github.com/leftthomas/SimCLR/blob/cee178b6cab1efab67f8b527a0cff91c9e793f5c/model.py#L27

**Comment by hendredlorentz** [comment]
tks! Have a nice day!
